documentation.html

```diff
@@ -34,7 +34,7 @@
         <main>
             <section class="component-section">
             
-                <h1 id="alert-component-heading">Alert</h1>
+                <h1 id="alert-component-heading" class="component-headings">Alert</h1>
             
                 <h3>Alerts are used to communicate a state that affects a system, feature or page.</h3>
```

Comment:
A `p` tag or `span` can be used here to wrap this text instead of a heading tag. 

Follow the [heading hierarchy](https://www.w3.org/WAI/tutorials/page-structure/headings/)

Reply:
Ok will change that.